bot.js

```diff
@@ -1,186 +1,111 @@
-// Core dependencies
 const Discord = require('discord.js');
 const request = require('request');
-const moment = require('moment');
-const momentDuration = require("moment-duration-format");
-
-// Commands
-const playingNow = require('./commands/playingnow.js');
-const games = require('./commands/games.js');
-const botHelp = require('./commands/bothelp.js');
-const userStatus = require('./commands/users.js');
-const stats = require('./commands/stats.js');
-const popcornGif = require('./commands/popcornGif.js');
-
-// Authorization
 const authDetails = require('./auth.json');
+const bot = new Discord.Client({ autoReconnect: true });
+const commands = require('./bot/commands.js');
 
-const prefix = '$';
-const adminPrefix = '-';
+// When the bot shuts down for whatever reason we post a msg in bottestchannel
+// to keep a log and notify bot admins.
+bot.on('disconnect', msg =>
```

Comment:
Is there a reason you moved this to the top?

I think we should keep these in some logical order. Maybe by the order the bot uses them or the frequency in which the bot uses them? Idea below:

1. bot.on(ready
2. bot.on(msg
3. bot.on(debug
4. bot.on(error
5. bot.on(disconnect

Reply:
I placed the event handlers in alphabetical order as a systematic approach to speed up the process of finding a specific function as the number of event handlers continues to grow.

Reply:
Rearranged event handlers as follows:

1. ready
2. message
3. guildMemberAdd
4. guildMemberRemove
5. disconnect
6. debug
7. warn
8. error